### PR TITLE
fix: Update columns flag to match help documentation

### PR DESCRIPTION
**Pull request by ghost**
:wave: Hey there, love the tool, really cool concept and easy carry with you binary.

The help documentation states the long flag for `-c` is `--columns` but when putting that on the command line I get the complaint that `--cols must be at least 1` So I've updated the flag in the main function to match the documentation.

Cheeers! :beers:

### CODE REVIEW

**Comment by eatonphil** [comment]
Hey! Thanks for noticing this! 

Actually what you did would be a breaking change so I think the better thing to do would be to just update the Readme instead.

Feel free to change this pr to do that or I can for you.

Thank you!!

**Comment by ghost** [comment]
:+1: Gotcha, thanks for letting me aware.

I've updated the help docs to match the main implementation.

I've left the line `Single binary CLI for generating a random schema of M columns to populate N rows of data` with the word columns because it's not the long-flag at that point yet. If you want me to change that as well let me know.

File: main.go [hunk 1]
```diff
@@ -402,7 +402,7 @@ Flags:
   -o, --out		Specify file to write to. If empty, writes to stdout
   -f, --format		Specify output format. Inferred by --out flag if present and --format flag empty
   -r, --rows		Number of rows to generate
-  -c, --columns 	Number of columns to generate
+  -c, --cols 	Number of columns to generate
```
Comment (eatonphil) [inline main.go]: Could you use tabs to make this line up with the rest? You may not see the difference in your editor but you can see the difference in Github's UI.

Comment (ghost) [reply main.go]: Oh you're right there was a space character there instead of a tab.

**Comment by eatonphil** [comment]
Awesome thank you!!

**Pull request merged**